src/js/view/controls/controlbar.js

```diff
@@ -527,6 +527,10 @@ export default class Controlbar {
             const dvrPosition = Math.min(this._model.get('position'), -1);
             const dvrSeekLimit = this._model.get('dvrSeekLimit');
 
+            this._model.once('seeked', () => {
```

Comment:
If you add this callback you then need to remove it in case the playlist item changes before seeking completes or in the case of an error.

Instead of doing that just call play. There is no need to wait for the callback. Calling `seek` and `play` (like the line you removed in the controller) results in playback after seeking.